app/controllers/workshops_controller.rb

```diff
@@ -11,13 +11,18 @@ def render_workshop
     begin
       workshop = params[:workshop]
       path = workshops_path.join(workshop, 'README.md')
+      url = request.url
 
       @title = workshop.humanize.titleize
 
       @metadata_page_name = @title
       @metadata_page_category = 'Workshop'
+      @metadata_do_not_track = signed_in?
```

Comment:
My reasoning:

- Having `@metadata_do_not_track` as the name for a boolean creates a double negative, which is confusing
- `@metadata_do_not_track` doesn't accurately describe its behavior. The user is still tracked when it's set to `true`, but we just don't send Segment the page view event.